guide/src/migration.md

```diff
@@ -105,9 +105,9 @@ Because there is no such distinction from Python, implementing these methods wil
 
 The PyO3 behavior in 0.16 has been changed to be closer to this Python behavior by default.
 
-### `wrap_pymodule!` now respects privacy correctly
+### `wrap_pymodule!` and `wrap_pyfunction!` now respect privacy correctly
```

Comment:
Isn't "respect" and "correctly" redundant? Meaning just "respect" would suffice or something "handle correctly"?

Reply:
I quite like having the full "respect privacy correctly" - IIRC I these things previously just used `pub(crate)`, so they did _some_ repecting of privacy but not correctly.